tools/coverage/githubUtil/github.go

```diff
@@ -21,13 +21,13 @@ import (
 )
 
 // convert filepath from profile format to github format
-// (github.com/$REPO_OWNER/$REPO_NAME/pkg/... -> pkg/...)
+// (knative.dev/$REPO_NAME/pkg/... -> pkg/...)
```

Comment:
not all repos are on `knative.dev` yet, this will break some other repos

Reply:
I'll change it to handle different logic